public/app/themes/clarity/src/globals/js/auth-heartbeat.js

```diff
@@ -1,6 +1,154 @@
+
 export default (function ($) {
-    // Send a request to the heartbeat endpoint, this will refresh the oauth token.
-    setInterval(function(){
-        $.get( "/auth/heartbeat" )
-    }, 10000)
-})(jQuery)
+    const Backdrop = {
+        cookie: {
+            name: 'moj_uat_session',
+            value: 'user_accepted',
+            set: (value, days = 7, path = '/') => {
+                const expires = new Date( Date.now() + days * 864e5).toUTCString()
+                document.cookie = Backdrop.cookie.name + '=' + value + '; expires=' + expires + '; path=' + path
+            },
+            get: () => {
+                return document.cookie.split('; ').reduce((r, v) => {
+                    const parts = v.split('=')
+                    return parts[0] === Backdrop.cookie.name ? parts[1] : r
+                }, '')
+            },
+            delete: () => Backdrop.cookie.set('', -1),
+        },
+        style: {
+            img: {
+                maxWidth: '120px',
+                height: 'auto',
+                float: 'left',
+                marginRight: '30px'
+            },
+            button: {
+                continue: {
+                    backgroundColor: 'green',
+                    color: 'white',
+                    cursor: 'pointer'
+                },
+                escape: {
+                    backgroundColor: '#2271b1',
+                    color: 'white',
+                    cursor: 'pointer'
+                }
+            },
+            modal: {
+                textAlign:'left',
+                padding:'20px 30px',
+                background:'#fff',
+                borderRadius:'5px',
+                alignItems: 'center',
+                verticalAlign:'middle',
+                maxWidth:'450px',
+                width: '450px',
+                margin:'50px auto',
+                border: '6px solid #2271b1'
+            },
+            backdrop: {
+                display:'flex',
+                alignItems: 'center',
+                backdropFilter: 'blur(5px)',
+                textAlign:'center',
+                position: 'fixed',
+                zIndex:1000000,
+                height:'100%',
+                width:'100%'
+            }
+        },
+        feature: {
+            can_run: () => {
+                const runOn = [
+                    'dev.intranet',
+                    'intranet.docker',
+                    'staging.intranet'
+                ];
+                let ii = 0;
+
+                for (ii; ii < runOn.length; ii++) {
+                    if (location.href.includes(runOn[ii])) {
+                        return true;
+                    }
+                }
+                return false;
+            }
+        },
+        modal: (title, html) => {
+            let modal = $('<div\>', { 'class': 'heartbeat__modal' })
```

Comment:
If you wanted to use semantic elements, `<dialog>` might work here.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog